tests/test_admin.py

```diff
@@ -524,3 +569,38 @@ class RegistrationTestCase(TestCase):
     def test_admin_is_registered(self):
         self.assertIn(PageContent, admin.site._registry)
         self.assertTrue(isinstance(admin.site._registry[PageContent], PageContentAdmin))
+
+
+@override_settings(CMS_PERMISSION=True)
+class CMSToolbarTestCase(CMSTestCase):
+    def test_pages_menu_item_url_has_no_params(self):
+        """
+        Create a page and get the toolbar for that page's preview
+        Then check that the page menu item does not have a query string,
+        so as not to trigger filters
+        """
+        user = self.get_superuser()
+        pagecontent = PageVersionFactory(content__template="")
+        toolbar = get_toolbar(
+            pagecontent.content,
+            user=user,
+            toolbar_class=PageAdminToolbar,
+            preview_mode=True,
+        )
+        toolbar.post_template_populate()
+        menu = toolbar.toolbar.get_menu("admin-menu")
+        pagemenu = menu.get_items()[0]
+        self.assertTrue("/en/admin/cms/pagecontent/?language=en", pagemenu.url)
```

Comment:
I'm pretty sure you're not interested in whether `"/en/admin/cms/pagecontent/?language=en"` evaluates to True. I presume this was meant to be `self.assertEqual`